PowerEditor/src/ScitillaComponent/FindReplaceDlg.h

```diff
@@ -410,15 +411,43 @@ class Progress
 	Progress(HINSTANCE hInst);
 	~Progress();
 
-	HWND open(HWND hOwner = NULL, const TCHAR* header = NULL);
-	bool isCancelled() const;
-	void setPercent(unsigned percent, const TCHAR *fileName) const;
-	void setInfo(const TCHAR *info) const {
+	HWND open(HWND hCallerWnd = NULL, const TCHAR* header = NULL);
+	void close();
+
+	bool isCancelled() const

```

Comment:
I wouldn't really worry about it if Notepad++ was smaller. It's just that Notepad++ is kinda big, and thus little things can greatly impact compilation time.

I guess it's ok here, but it still makes me grumble >:(